modules/util/path.go

```diff
@@ -150,3 +153,23 @@ func StatDir(rootPath string, includeDir ...bool) ([]string, error) {
 	}
 	return statDir(rootPath, "", isIncludeDir, false, false)
 }
+
+// FileURLToPath extracts the path informations from a file://... url.
+func FileURLToPath(u *url.URL) (string, error) {
+	if u.Scheme != "file" {
+		return "", errors.New("URL scheme is not 'file': " + u.String())
+	}
+
+	path := u.Path
+
+	if runtime.GOOS != "windows" {
+		return path, nil
+	}
+
+	// If it looks like there's a Windows drive letter at the beginning, strip off the leading slash.
+	re := regexp.MustCompile("/[A-Za-z]:/")
+	if re.MatchString(path) {
+		return path[1:], nil
```

Comment:
Not sure I understand the purpose of this function but wouldn't it be more correct to keep drive letter and just convert forward slashes to backslashes, like this node function does?

https://nodejs.org/api/url.html#url_url_fileurltopath_url

Reply:
It doesn't remove the drive letter. The lines convert `/C:/` to `C:/`.

Reply:
Yeah it's another ideocrazy of golang it seems.